src/components/Dasboard/styles.tsx

```diff
@@ -11,11 +14,19 @@ export const useStyles = makeStyles((theme: Theme) =>
 			background: theme.palette.primary.main,
 		},
 		leftPanelContent: {},
+		leftPanelActiveLink: {
+			background: "white",
```

Comment:
Can we use colors from the colors package provided by material ui or from theme object ?

Reply:
Done